peps/api/index.rst

```diff
@@ -0,0 +1,64 @@
+PEPs API
+========
+
+There is a read-only API of published PEPs available at:
+
+* https://peps.python.org/api/peps.json
+
+The structure is like:
+
+.. code-block:: json
```

Comment:
Perhaps we can include a pseudo-schema?
```suggestion
The structure is like:

.. code-block:: json

   {
     "<PEP number>": {
       "number": integer,
       "title": string,
       "authors": string,
       "discussions_to": string | null,
       "status": "Accepted" | "Active" | "Deferred" | "Draft" | "Final" | "Provisional" | "Rejected" | "Superseded" | "Withdrawn",
       "type": "Informational" | "Process" | "Standards Track",
       "topic": "governance" | "packaging" | "release" | "typing" | "",
       "created": string,
       "python_version": string | null,
       "post_history": string,
       "resolution": string | null,
       "requires": string | null,
       "replaces": string | null,
       "superseded_by": string | null,
       "url": string
     },

Date values are formatted as DD-MMM-YYYY,
and multiple dates are combied in a comma-separated list.

For example:

.. code-block:: json
```

Reply:
Hmm, this fails to build due to unquoted strings:

> /home/runner/work/peps/peps/peps/api/index.rst:10: WARNING: Lexing literal_block '{\n  "<PEP number>": {\n    "number": integer,\n    "title": string,\n    "authors": string,\n    "discussions_to": string | null,\n    "status": "Accepted" | "Active" | "Deferred" | "Draft" | "Final" | "Provisional" | "Rejected" | "Superseded" | "Withdrawn",\n    "type": "Informational" | "Process" | "Standards Track",\n    "topic": "governance" | "packaging" | "release" | "typing" | "",\n    "created": string,\n    "python_version": string | null,\n    "post_history": string,\n    "resolution": string | null,\n    "requires": string | null,\n    "replaces": string | null,\n    "superseded_by": string | null,\n    "url": string\n  },' as "json" resulted in an error at token: 'i'. Retrying in relaxed mode.

Suggestions?

Reply:
Would it parse as JavaScript? I think that looks like syntactically valid JS.

Reply:
Good thinking! Yes, it does. The colours are different, but I don't think that's a big problem:

<img width="811" alt="image" src="https://github.com/user-attachments/assets/9400e5e7-e89b-4bf1-9c8b-8fa2f676cc3e">

Reply:
LGTM!